### DISCUSSION TITLE
Hanging systemd-binfmt.service on Arch

### DISCUSSION THREAD

**Discussion by rwilliams** [arkane-systems/genie#180] [Q&A]
After upgrading to Genie 1.31+ I starting experiencing the `Waiting for systemd....!!!!!!!!!!!` timeout. It looks like the culprit for me is `systemd-binfmt.service` and when I run `journalctl -u systemd-binfmt.service` I get the following. 

```
Jul 09 09:13:15 ArchWSL-wsl systemd-binfmt[32]: Failed to add binary format: Too many levels of symbolic links
Jul 09 09:13:15 ArchWSL-wsl systemd[1]: systemd-binfmt.service: Main process exited, code=exited, status=1/FAILURE
Jul 09 09:13:15 ArchWSL-wsl systemd[1]: systemd-binfmt.service: Failed with result 'exit-code'.
Jul 09 09:13:15 ArchWSL-wsl systemd[1]: Failed to start Set Up Additional Binary Formats.
```

I was wondering if any other Arch users of Genie have run into this or if anyone has any ideas on fixing it?

Thanks!

**Comment by cerebrate** [comment]
See discussion under #142 for the current state of this.
